prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -83,17 +83,35 @@ class FHIRRouter(
             "%specimen.collection.collected.exists()" +
             ") or " +
             "%serviceRequest.occurrence.exists() or " +
-            "%observation.effective.exists())"
+            "%observation.effective.exists())",
     )
 
     /**
-     * Default Rule:
+     * Default Rules for quality filter on ETOR_TI topic:
+     *   Must have message ID
+     */
+    private val etorTiQualityFilterDefault: ReportStreamFilter = listOf(
+        "%messageId.exists()",
+    )
+
+    val qualityFilterDefaults = mapOf(
+        Pair(Topic.FULL_ELR, fullElrQualityFilterDefault),
+        Pair(Topic.ETOR_TI, etorTiQualityFilterDefault),
+    )
+
+    /**
+     * Default Rule (used for ETOR_TI and FULL_ELR):
      *  Must have a processing mode id of 'P'
      */
-    val processingModeFilterDefault: ReportStreamFilter = listOf(
+    private val processingModeFilterDefault: ReportStreamFilter = listOf(
         "%processingId.exists() and %processingId = 'P'"
     )
 
+    val processingModeDefaults = mapOf(
```

Comment:
Like above, just a lil comment for clarity

Reply:
Added!